### ISSUE TITLE
GetQuotes not working

### ISSUE THREAD

**Issue by smjure** [d3an/finviz#77]
Hey there, thanks for the great api. 

I'd like to get e.g. a particular's ticker `Current Ratio`. Based on the screener example on your home page, If I do e.g.
```	
client := quote.New(nil)
stocks, err := client.GetQuotes([]string{"AAPL", "TSLA"})
```

all parameters/columns yield no information, please see the pic:

![2021-12-27_10-35](https://user-images.githubusercontent.com/22912191/147458201-379359a6-748c-405b-bd43-0d4d894f7ce8.png)

Am I doing sth wrong? Any suggestion appreciated :+1:

**Comment by d3an** [comment]
Looking into it. Need to update the scraper.

**Comment by d3an** [comment]
Fixed in [v1.0.8](https://github.com/d3an/finviz/releases/tag/v1.0.8).
